components/question/Dropdown/dropdownStyles.js

```diff
@@ -0,0 +1,40 @@
+export const dropdownStyles = {
+  option: (provided, optionProps) => ({
+    ...provided,
+    background: optionProps.isFocused || optionProps.isSelected ? 'grey' : 'inherit',
+  }),
+  control: provided => ({
+    ...provided,
+    height: '3.5rem',
+    borderRadius: '0.75rem',
+    backgroundColor: 'rgba(255, 255, 255, 0.18)',
+    borderColor: 'transparent',
+    boxShadow: 'none',
+    ':hover': { borderColor: 'transparent' },
```

Comment:
Can we set the `font-size` and `line-height` for the text inside the select? 💭

Reply:
sure, done in 5cd98327eaba4fe6817334960dd6986d24351bc7, Thanks!